### PR TITLE
Update accordion tests

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1044

## Description
 - Updated the accordion component test files

### CODE REVIEW

File: src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx [hunk 1]
```diff
@@ -43,15 +43,15 @@ describe('AccordionItemPanel', () => {
...
       expect(
-        wrapper
-          .find(AccordionItemPanel)
-          .find('div')
-          .hasClass('accordionPanelDefault')
-      ).toBe(true);
+        container
+          .querySelector('.accordionDefault')
+          ?.querySelector('div')
+          ?.querySelector('.accordionPanelDefault')
```
Comment (imransl) [inline src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx:49]: Can't this be written with just one `querySelector` like this?

```.querySelector('.accordionDefault .accordionPanelDefault')```

File: src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx [hunk 2]
```diff
@@ -60,27 +60,34 @@ describe('AccordionItemPanel', () => {
...
+      ).toBeTruthy();
+
+      expect(
+        container
+          .querySelector('.accordionDefault')
+          ?.querySelector('div')
+          ?.querySelector('.accordionPanelDefault')
+          ?.classList.contains('foo')
+      ).toBeTruthy();
     });
```
Comment (imransl) [inline src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx:76]: Same as the comment I mentioned above about `querySelector`. If we can do it with just one, then why not?

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx [hunk 3]
```diff
@@ -152,18 +180,29 @@ describe('Accordion', () => {
...
+        userEvent.click(
+          container.querySelector(
+            '.accordionButtonDefault'
+          ) as HTMLButtonElement
+        );
+        userEvent.click(
+          container.querySelector(
+            '.accordionButtonDefault'
+          ) as HTMLButtonElement
+        );
```
Comment (azangru) [inline src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx:192]: It's clicking the same element two times, isn't it? And is also reusing this element below? Wouldn't the intention be clearer if the result of the `querySelector` method were assigned to a variable?

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx [hunk 4]
```diff
@@ -190,15 +232,18 @@ describe('Accordion', () => {
...
+          container
+            .querySelector('.accordionButtonDefault')
+            ?.closest('div')
+            ?.getAttribute('aria-expanded')
+        ).toEqual('false');
       });
     });
 
     describe('onChange prop', () => {
       it('is invoked with an array of expanded items’ uuids, if there are any', async () => {
```
Comment (azangru) [inline src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx:244]: Any reason for this function to be async anymore?

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx [hunk 5]
```diff
@@ -208,17 +253,18 @@ describe('Accordion', () => {
...
+        userEvent.click(
+          container.querySelector(
+            '.accordionButtonDefault'
+          ) as HTMLButtonElement
+        );
 
         expect(onChange).toHaveBeenCalledWith([UUIDS.FOO]);
       });
 
       it('is invoked with an empty array, if no items are expanded', async () => {
```
Comment (azangru) [inline src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx:265]: Same here re the async function

File: src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx (hunk 1 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx]: That's done

File: src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx (hunk 2 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx:68]: That's done

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx (hunk 3 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx]: That's done

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx (hunk 4 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx]: No it doesn't have to be async. I've removed it

File: src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx (hunk 5 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx]: That's done

**Review by azangru — APPROVED** [review]

**Pull request merged**